deluge/ui/gtkui/piecesbar.py

```diff
@@ -55,9 +50,13 @@ def __init__(self):
         self.__cr = None
 
         self.connect('size-allocate', self.do_size_allocate_event)
-        self.set_colormap(gtk.gdk.colormap_get_system())
+        self.connect("draw", self.expose)
+        self.set_visual(Gdk.Visual.get_best())
         self.show()
 
+    def expose(self, widget, event):
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://www.codacy.com/app/tbkizle/deluge/file/2628318271/issues/source?bid=3238201&fileBranchId=3238205#l57)